tests/sqlparser_common.rs

```diff
@@ -12440,3 +12441,83 @@ fn test_reserved_keywords_for_identifiers() {
     let sql = "SELECT MAX(interval) FROM tbl";
     dialects.parse_sql_statements(sql).unwrap();
 }
+
+#[test]
+fn parse_create_table_with_enum_types() {
+    let sql = "CREATE TABLE t0 (foo ENUM8('a' = 1, 'b' = 2), bar ENUM16('a' = 1, 'b' = 2), baz ENUM('a', 'b'))";
+    match all_dialects().verified_stmt(sql) {
+        Statement::CreateTable(CreateTable { name, columns, .. }) => {
+            std::assert_eq!(name.to_string(), "t0");
+            std::assert_eq!(
```

Comment:
oh for the asserts was the `std::assert_eq` auto imported or was it a need to use them specifically for the tests?

Reply:
Yes, `std::` is unnecessary here. 👍🏿